### PR TITLE
added instructions for JavaScript using jasmine to project

### PR DESCRIPTION
**Pull request by Steve-code16**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/projects/oop/animals/part2/_index.md [hunk 1]
```diff
@@ -2,19 +2,19 @@
 _db_id: 225
 content_type: project
 flavours:
-- any_language
+  - any_language
```
Comment (ry-oc) [inline content/projects/oop/animals/part2/_index.md]: please can you remove the added spaces in the front matter?

File: content/projects/oop/animals/part2/_index.md [hunk 2]
```diff
@@ -2,19 +2,19 @@
...
 from_repo: projects/oop/animals/part1
 prerequisites:
   hard:
-  - projects/tdd/simple-calculator-part1
-  - projects/oop/animals/part1
+    - projects/tdd/simple-calculator-part1
+    - projects/oop/animals/part1
```
Comment (ry-oc) [inline content/projects/oop/animals/part2/_index.md]: please can you remove the added spaces in the front matter?

File: content/projects/oop/animals/part2/_index.md [hunk 3]
```diff
@@ -2,19 +2,19 @@
...
   soft: []
 ready: true
 story_points: 3
 submission_type: continue_repo
 tags:
-- unit-testing
-- oop
+  - unit-testing
+  - oop
```
Comment (ry-oc) [inline content/projects/oop/animals/part2/_index.md]: please can you remove the added spaces in the front matter?

**Review by ry-oc — CHANGES_REQUESTED** [review]
Looks good, but there was some spacing added automatically that needs to be corrected.

Thanks for your contribution

**Review by Vuyisanani — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**